packages/jellyfish-api-core/src/category/poolpair.ts

```diff
@@ -202,19 +221,14 @@ export interface AddPoolLiquiditySource {
 }
 
 export interface AddPoolLiquidityOptions {
```

Comment:
`AddPoolLiquidityOptions` and `RemovePoolLiquidityOptions` interfaces have same variable. They should be `PoolLiquidityOptions`